src/MeetingsSDKAdapter.test.js

```diff
@@ -340,18 +332,15 @@ describe('Meetings SDK Adapter', () => {
 
   describe('stopStream()', () => {
     const mockStop = jest.fn();
-    let mockMediaStreamInstance;
 
     beforeEach(() => {
-      mockMediaStreamInstance = {
-        getTracks: () => [{stop: mockStop}],
-      };
-
-      global.MediaStream = jest.fn(() => mockMediaStreamInstance);
+      for (const track of mockSDKMediaStreams.localAudio.getTracks()) {
+        track.stop = mockStop;
```

Comment:
I think you can still define the `stop()` function in the mock MediaStream object and avoid matching it by making it not enumerable:

```js
track.defineProperty(
   value: jest.fn(),
   enumerable: false,
);